src/main/java/com/akathist/maven/plugins/launch4j/Launch4jMojo.java

```diff
@@ -456,6 +462,36 @@ private void setPermissions(File workdir) {
     }
 
     /**
+     * If custom header objects or libraries shall be linked, they need to sit inside the launch4j working dir.
```

Comment:
Sure. I've tested a configuration like this:
```
<objs>
	<obj>w32api/crt2.o</obj>
	<obj>${basedir}/src/main/native/copy_of_guihead.o</obj>
	<obj>src/main/native/copy_of_head.o</obj>
</objs>
```
I copied the default header objects from the launch4j sources into my project dir and renamed them. Also, I linked  w32api/crt2.0 which is already part of the working directory and not included into my project sources. This will produce a running launcher even if an absolute path is specified by using ${basedir}.

Of course, I made sure nothing will break if header objects aren't used at all.